cadc-util/src/test/java/ca/nrc/cadc/util/RSASignatureGeneratorValidatorTest.java

```diff
@@ -77,6 +80,30 @@ public void testSignVerify() throws Exception {
         }
     }
 
+    @Test
+    public void testSignVerifyBytes() throws Exception {
+        final String testString = "cadcauthtest1-" + new Date();
```

Comment:
That new static method above would be used here (inside for loop) and encapsulate some of this code